charts/partners/sapiens/idit-runtime-umbrella-chart/1.0.18/report.yaml

```
@@ -0,0 +1,128 @@
+apiversion: v1
+kind: verify-report
+metadata:
+    tool:
+        verifier-version: 1.12.1
+        profile:
+            VendorType: partner
+            version: v1.2
+        reportDigest: uint64:10800224655124558046
+        chart-uri: https://idit-artifactory.sapiens.com:443/artifactory/idit-devops-local/chart/idit-runtime/1.0.18/idit-runtime-1.0.18.tgz
+        digests:
+            chart: sha256:4f454ed42420a2b89a4ff3c265415556c1c3bc244fe94522eefa295b33c0cc7b
+            package: 1af6f76aa182df2e0b3436aaff8cefaecd950b7c2d1712b7a9e8faf6e5f4ce73
+        lastCertifiedTimestamp: "2023-08-28T12:44:39.471229+00:00"
+        testedOpenShiftVersion: "4.10"
+        supportedOpenShiftVersions: 4.10 - 4.13
+        webCatalogOnly: false
+    chart:
+        name: idit-runtime
+        home: ""
+        sources: []
+        version: 1.0.18
+        description: A Helm chart for Kubernetes
+        keywords: []
+        maintainers: []
+        icon: ""
+        apiversion: v2
+        condition: ""
+        tags: ""
+        appversion: 1.16.0
+        deprecated: false
+        annotations:
+            app.idit/version: 19.2.0.2
+            charts.openshift.io/name: IDITSuite 18.
+        kubeversion: '>= 1.23.0 <= 1.26.3'
```

This needs to look like this.

```suggestion
        kubeversion: '>=1.23.0 <=1.26.3'
```

This fix needs to go into your Chart.yaml, and then the report needs to be regenerated using chart-verifier. Submit the updated report, and I think we should get further into the certification process.

I've created an issue to look more closely at our version interpreting logic in the pipeline, but fixing the above should resolve this for you more quickly.

---

OK, will try and check